src/models/Expense.model.js

```diff
@@ -1,9 +1,50 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+
+    spentAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+
+    amount: {
+      type: DataTypes.INTEGER,
```

Comment:
Consider using `DataTypes.DECIMAL` or `DataTypes.FLOAT` for the `amount` field to better handle monetary values, especially if you need to account for cents.